plugins/playlist-backend/src/service/router.ts

```diff
@@ -85,7 +85,7 @@ export async function createRouter(
       request.header('authorization'),
     );
 
-    const user = await identity.getIdentity({ request });
+    const user = await identity.getIdentity({ request, optional: true });
```

Comment:
This comment is pedantic and can be happily ignored.

Im wondering if we should decide on a name to call the response from this call and use it across the other plugins too.

We are already using: user, userIdentity, callerIdentity and identity.